src/doc_types.h

```diff
@@ -13,8 +14,7 @@ extern "C" {
 static inline DocumentType getDocType(RedisModuleKey *key) {
   if (RedisModule_KeyType(key) == REDISMODULE_KEYTYPE_HASH) {
     return DocumentType_Hash;
-  } else if (1) {
-    // TODO: fix once can figure out JSON
+  } else if (japi->isJSON(key)) {
```

Comment:
If reachable also when rejson module is not loaded, need to first test `japi` is not null